### DISCUSSION TITLE
빌드 에러 : System.Exception: timeout

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#642] [Issue Report (Legacy)]
월드 1.2.0 

어제까지만 해도 잘되던 빌드가 안되네요...
플레이는 잘 됩니다...

새 프로젝트 만들어서 테스트 해봤더니 거기서는 빌드가 잘 되드라구요 

Library, Logs, Temp 폴더 다 지우고 재시작 해봤습니다 

Reimport ALL 도 해봤습니다 

혹시 빌드시 반드시 필요한 파일들이 있나요? 

svn사용중인데요 assets, packages, projectsettings 폴더만 빼고 다 ignore되어 있어요 

이것이 문제가 될수 있을까요?

버전 입니다 
![image](https://user-images.githubusercontent.com/50188795/164148166-0d36553e-6006-4127-b89b-c1f1187004dd.png)

로그인 잘되어 있구요 
![image](https://user-images.githubusercontent.com/50188795/164148369-951f38f1-faf2-4d12-948b-b29731aa2ef0.png)

월드 셋팅 잘되어 있어요 
![image](https://user-images.githubusercontent.com/50188795/164148389-98f965f3-7117-43fd-ba61-9bd5c60d7689.png)

플랫폼 설정 입니다 
![image](https://user-images.githubusercontent.com/50188795/164148415-bc84e820-74bd-4cad-a1f7-39f774ea05ac.png)

Environment설정 입니다 
![image](https://user-images.githubusercontent.com/50188795/164148439-5aac1efb-dbb6-4e40-8f27-49f7cafec1d3.png)

빌드 셋팅 입니다 
![image](https://user-images.githubusercontent.com/50188795/164148871-230b54d0-9f69-441c-9ede-12fbfcc09e0b.png)

![image](https://user-images.githubusercontent.com/50188795/164148265-6ab5d9b2-9abc-4011-88dc-dd1d5a9c2fed.png)
![image](https://user-images.githubusercontent.com/50188795/164148274-c5cbe399-f566-47ff-88ff-3ec9767df93f.png)
![image](https://user-images.githubusercontent.com/50188795/164148287-eaeba4cd-1394-4c32-8497-8c56dbbd902d.png)
![image](https://user-images.githubusercontent.com/50188795/164148303-d5e5c901-166f-442c-a732-ff5ad58f1715.png)
![image](https://user-images.githubusercontent.com/50188795/164148324-06afffe5-d1f8-4a3f-8597-c990e8780a11.png)

무엇이 문제일까요...?!

도와주세요 ~

**Comment by smilejsu82** [comment]
이유는 모르겠지만 코드상에 Debug.DrawRay 부분을 주석 처리 했더니 빌드 되었습니다

**Reply by x4cf** [reply]
실수로라도 기즈모 함수를 넣은 분들이 조금 더 쉽게 오류를 찾기 좋아보이는군요!
